getting-started/README.md

```diff
@@ -10,11 +10,10 @@ But a few differences [do exist](https://community.optimism.io/docs/developers/b
 
 ## Optimism endpoint URL
 
-To access any Ethereum type network you need an endpoint. There are several ways to get one:a free tier for low usage.
-
-1. For *limited* development use, [Optimism-provided endpoints](https://community.optimism.io/docs/useful-tools/networks/). 
-   Note that these endpoints are rate limited, so they are not for use in QA or production environments.
+To access any Ethereum type network you need an endpoint. 
+We recommend you get one from [Alchemy, our preferred provider](https://www.alchemy.com/).
 
+If you prefer, we have [other providers too](https://community.optimism.io/docs/useful-tools/providers/).
```

Comment:
You already said prefer, maybe say here 

Alternatively, we have [other great providers] that support our networks.